CONTRIBUTING.md

```diff
@@ -0,0 +1,181 @@
+
+Put the reported case in the tests, then run integration tests with the following command:
+```bash
+ansible-test integration test_mysql_user --docker -vvv > ~/test.log
+```
+If the tests do not want to run, first, check you complete step 3 of this guide.
+
+If the tests ran successfully, there are usually two possible outcomes:
+a) If the bug has not appeared and the tests have passed successfully, ask the reporter to provide more details. The bug can be not a bug actually or can relate to a particular software version used or specifics of local environment configuration.
+
+b) The bug has appeared and the tests has failed as expected showing the reported symptoms.
+
+15. Fix the bug.
+
+16. Run ``flake8``:
+```bash
+flake8 /path/to/changed/file
+```
+It is worth installing and running and ``flake8`` against the changed file(s) first.
+It shows unused imports, which is not shown by sanity tests, as well as other common issues.
+
+17. Run sanity tests:
+```bash
+ansible-test sanity /path/to/changed/file --docker
+```
+If they failed, look at the output carefully - it is usually very informative and helps to identify a problem line quickly.
+Sanity failings usually relate to wrong code and documentation formatting.
+
+18. Run integration tests:
+```bash
+ansible-test integration test_mysql_user --docker -vvv > ~/test.log
+```
+
+There are two possible outcomes:
+a) They have failed. Look into the ``test.log``.
+Errors are usually at the end of the file.
+Fix the problem place in the code and run again.
+Repeat the cycle until the tests pass.
+
+b) They have passed. Remember they failed originally? Our congratulations! You has probably fixed the bug, though we hope not introducing a couple of new ones;)
+
+19. Commit your changes with an informative but short commit message:
+```bash
+git add /path/to/changed/file
+git commit -m "mysql_user: fix crash when ..."
+```
+
+20. Push the branch to the ``origin`` (your fork):
+```bash
+git push origin name_of_my_branch
+```
+
+21. Go to the ``upstream`` (http://github.com/ansible-collections/community.mysql.git
+
+22. Go to ``Pull requests`` tab and create a pull request.
+
+GitHub is tracking your fork, so it should see the new branch in it and automatically offer
+to create a pull request. Sometimes GitHub does not do it and you should click the ``New pull request`` button yourself.
+Then choose ``compare across forks`` under the ``Compare changes`` title.
+Choose your repository and the new branch you pushed in the right drop-down list.
+Confirm. Fill out the pull request template with all information you want to mention.
+Put "Fixes + link to the issue" in the pull request's description.
+Put "[WIP] + short description" in the pull request's title.
+Click ``Create pull request``.
+
+23. Add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs). It will be published in release notes, so users will know about the fix.
```

Comment:
```suggestion
23. Add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs). It will be published in release notes, so users will know about the fix.
```

Reply:
Makes sense!